src/database/migrations/1732279680865-LocationColumnInEventAndGroup.ts

```diff
@@ -0,0 +1,33 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddLocationPointToEventAndGroup1732279680865
+  implements MigrationInterface
+{
+  name = 'AddLocationPointToEventAndGroup1732279680865';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const schema = queryRunner.connection.options.name || 'public';
+    // Adding locationPoint column to "groups"
+    await queryRunner.query(
+      `ALTER TABLE "${schema}"."groups" ADD "locationPoint" geography(Point,4326)`,
```

Comment:
Should there also be an index for that field? Please place in BaseTables instead.